src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -72,17 +72,24 @@ class Tabulator {
   // tracks residual surplus from multi-seat contest vote transfers
   private final Map<Integer, BigDecimal> roundToResidualSurplus = new HashMap<>();
   // precincts which may appear in the cast vote records
-  private final Set<String> precinctNames;
+  private final Set<String> precinctNames = new HashSet<>();
```

Comment:
This seems to be the only place in the codebase where we call this `precinctNames` instead of `precinctIds`. Is there a reason for this? If not, let's make it consistent.

Reply:
Sure, I renamed all instances of `precinctName` to `precinctId`